This patch has been integrated into Glance 11.0.0 in Liberty but has
not yet been back-ported to Kilo.

commit b7fb5bf0f89f657ead98024ee0168f2c2fa7a776
Author: Erno Kuvaja <jokke@usr.fi>
Date:   Mon Sep 28 10:44:38 2015 +0000

    Return missing authtoken options
    
    Example configs were missing keystone_authtoken section after moving
    to generated config files. This change returns that to generation.
    
    Closes-Bug: #1500361
    
    Change-Id: I6ee82c38061d483cea7254d155d9a72436880e84
    (cherry picked from commit b1d2d938d282ccd51986e57a638f8ea5bec56b0f)

diff --git a/etc/oslo-config-generator/glance-api.conf b/etc/oslo-config-generator/glance-api.conf
index d60dc12..3f24718 100644
--- a/etc/oslo-config-generator/glance-api.conf
+++ b/etc/oslo-config-generator/glance-api.conf
@@ -7,5 +7,5 @@ namespace = oslo.messaging
 namespace = oslo.db
 namespace = oslo.db.concurrency
 namespace = oslo.policy
-namespace = keystoneclient.middleware.auth_token
+namespace = keystonemiddleware.auth_token
 namespace = oslo.log
diff --git a/etc/oslo-config-generator/glance-registry.conf b/etc/oslo-config-generator/glance-registry.conf
index 0663fc1..961abd9 100644
--- a/etc/oslo-config-generator/glance-registry.conf
+++ b/etc/oslo-config-generator/glance-registry.conf
@@ -6,5 +6,5 @@ namespace = oslo.messaging
 namespace = oslo.db
 namespace = oslo.db.concurrency
 namespace = oslo.policy
-namespace = keystoneclient.middleware.auth_token
+namespace = keystonemiddleware.auth_token
 namespace = oslo.log
